docs/index.md

```diff
@@ -2664,6 +2664,50 @@ Some more examples:
 
 Label sets are helpful for organizing and filtering large spec suites in which different specs satisfy multiple overlapping concerns.  The use of label set filters is intended to be a more powerful and expressive alternative to the use of regular expressions.  If you find yourself using a regular expression, consider if you should be using a label set instead.
 
+##### Version-Based Label Filtering
+
+Based on the Label sets feature, Ginkgo also supports version-based label filtering. This allows you to mark tests with version constraints using Label sets.
+
+```go
+Describe("Feature with version requirements", Label("minVersion:v1.2.3"), Label("maxVersion:v2.2.3"), func() {
+  It("should only run when v1.2.3 <= version <= v2.2.3", func() {
+    // This test will only run when the version constraint is satisfied
+  })
+
+  It("should only run when v1.2.3 <= version <= v2.0.0", Label("maxVersion:v2.0.0"), func() {
+    // This test will only run when the version constraint is satisfied
+  })
+})
+
+It("min version test", Label("minVersion:v2.0.0"), func() {
+  // Test code here
+})
+```
+
+Tests can be labeled with `minVersion` and `maxVersion` to specify the version range in which they are applicable.
+
+- `minVersion:vX.Y.Z`: Specifies the minimum version required for the test to run.
+- `maxVersion:vA.B.C`: Specifies the maximum version up to which the test should run.
```

Comment:
For the implementation, should we just use the hardcoded string `minVersion` and `maxVersion` for parsing or should it still be customized by the user? I'm talking about this because we need to think about an efficient way to parse the version, otherwise parse all of the labels would cause performance abuse.